axon-server-connector/src/main/java/org/axonframework/axonserver/connector/query/AxonServerQueryBus.java

```diff
@@ -326,13 +333,13 @@ public void handle(QueryRequest query,
 
         io.axoniq.axonserver.connector.query.SubscriptionQueryResult result =
                 axonServerConnectionManager.getConnection(targetContext)
-                                           .queryChannel()
-                                           .subscriptionQuery(
-                                                   subscriptionSerializer.serializeQuery(interceptedQuery),
-                                                   subscriptionSerializer.serializeUpdateType(interceptedQuery),
-                                                   configuration.getQueryFlowControl().getInitialNrOfPermits(),
-                                                   configuration.getQueryFlowControl().getNrOfNewPermits()
-                                           );
+                        .queryChannel()
```

Comment:
Guessing that after setting the `axon_code_style.xml`, these changes would be reverted.